Cargo.toml

```diff
@@ -32,5 +31,5 @@ futures-lite = "2"
 profiling = "1"
 puffin_http = "0.16.1"
 tracy-client = "0.18"
-wgpu = { version = "24.0.0", default-features = true }
```

Comment:
let's mark this explicitly `25.0.1` since this required in the project

Reply:
There isn’t a 25.0.1 though for `wgpu` itself.